test/verify/check-metrics

```diff
@@ -115,7 +115,7 @@ class TestMetrics(MachineCase):
         # check whether we see that in the plot.
         #
         meminfo = read_mem_info(m)
-        mem_used = meminfo['MemTotal'] - meminfo['MemFree']
+        mem_used = meminfo['MemTotal'] - meminfo['MemAvailable']
```

Comment:
(Should this be in the previous commit?)

Reply:
oh yea, good spot.